packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1055,12 +1057,22 @@ export class ColorPicker
   //
   //--------------------------------------------------------------------------
 
+  handleChannelInputChange = (event: Event): void => {
+    const customEvent = event as CustomEvent;
+    this.handleChannelChange(customEvent);
+  };
+
   handleKeyDown(event: KeyboardEvent): void {
     if (event.key === "Enter") {
       event.preventDefault();
     }
   }
 
+  handleInputFocus = (event: Event): void => {
```

Comment:
Selecting text is a great enhancement, but it does seem unrelated to committing on paste/input. Could we submit this separately?

Reply:
I'll create a separate issue for this.